src/TLuaInterpreter.cpp

```diff
@@ -5869,6 +5869,37 @@ int TLuaInterpreter::tempButton(lua_State* L)
     return 1;
 }
 
+int TLuaInterpreter::setButtonStyleSheet(lua_State* L)
+{
+    //args: name, css text
+    QString name, css;
+
+    if (!lua_isstring(L, 1)) {
+        lua_pushstring(L, "setButtonStyleSheet: wrong first arg");
+        lua_error(L);
+        return 1;
+    } else {
+        name = lua_tostring(L, 1);
```

Comment:
Use `QString::fromUtf8(lua_tostring(L, 1))` to capture - in 4.0 we'll have full i18n support so we're working towards transitioning all data capture to allow utf8.